regression/tests/studio/test_course_pages.py

```diff
@@ -0,0 +1,54 @@
+"""
+Course pages test
+"""
+from uuid import uuid4
+from bok_choy.web_app_test import WebAppTest
+
+from regression.pages.studio.login_studio import StudioLogin
+from regression.tests.helpers import LoginHelper, get_course_info
+from regression.pages.studio.pages_page_studio import PagesPageExtended
+
+
+class CoursePagesTest(WebAppTest):
+    """
+    Course Pages test
+    """
+    def setUp(self):
+        super(CoursePagesTest, self).setUp()
+        self.course_info = get_course_info()
+        self.login_page = StudioLogin(self.browser)
+        LoginHelper.login(self.login_page)
+        self.pages_page = PagesPageExtended(
+            self.browser,
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run']
+        )
+
+    def test_pages(self):
+        """
+        Tests basic scenarios related to pages section of course.
+        Scenarios covered are adding, editing and
+        deleting pages.
+        """
+        self.pages_page.visit()
+        # Get total count of pages currently present.
+        initial_page_count = self.pages_page.get_page_count()
+        # Add a new page.
+        self.pages_page.add_page()
+        # Verify that the initial and current page count are not the same.
+        self.assertNotEqual(
+            initial_page_count, self.pages_page.get_page_count)
```

Comment:
Is page count sufficient to check if a new page is added successfully? 

I think assert should be improved. Lets say `initial_page_count` is 1 and after`add_page` new count is 0 for whatever reason. In this case it will pass but it shouldn't.